tests/peer_did_creator_tests.py

```diff
@@ -0,0 +1,10 @@
+import pytest
+
+
+class TestPeerDIDCreator:
```

Comment:
No need to define a class for tests. you can write `def test_create_numalgo_0` right at the top level of the .py file.
Please name the file test_peer_did_creator (test at the beginning)